### DISCUSSION TITLE
Interesting results with 231-instruct-pix2pix-image-editing

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#806] [Show and tell]
![lake_tahoe](https://user-images.githubusercontent.com/1720147/216252633-0f081af0-d0ed-4781-af2b-1c494bc89cd2.jpg)

What can we come up with? :)

![image](https://user-images.githubusercontent.com/1720147/216252872-c365ba83-da0c-4701-aea4-02761847df84.png)
![image](https://user-images.githubusercontent.com/1720147/216252935-246ffc2d-cc19-4230-9e31-e0cdf15e289f.png)

**Comment by raymondlo84** [comment]
![horses](https://user-images.githubusercontent.com/1720147/216255487-9bbfc540-e8cc-42cf-ae81-a9ab9adda062.JPG)
![image](https://user-images.githubusercontent.com/1720147/216254959-698a5c94-0c7d-4c09-bbca-bba4966f16a6.png)

**Reply by AnishaUdayakumar** [reply]
![image](https://user-images.githubusercontent.com/109281183/216256966-f1d143a8-f489-4127-8683-32e7a43ba5e4.png)

**Comment by openvino-dev-samples** [comment]
Shall we run a AI painting contest. Let developer play with notebook, and output their results here.  The best one will get a gift, lol

**Reply by raymondlo84** [reply]
Do it :)

**Reply by raymondlo84** [reply]
Zhuo/Adrian got hoodies and we can make it happen.

**Reply by raymondlo84** [reply]
Can you also test these notebooks, it uses LOTs of RAM and on Ubuntu it is very easy to crash if people didn't define a large enough SWAP partition.

**Reply by openvino-dev-samples** [reply]
I got 32GB RAM on my ubuntu, no issue on crash.
